src/routes/podcasts.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
```

Comment:
suggestion: I propose we treat `routes` as being on the `app` layer of FSD. That said, ideally we'd want to keep as little page-based responsibility here. Perhaps we could move all this code to `pages/podcast-episodes/ui`?